gradio/blocks.py

```diff
@@ -310,8 +312,91 @@ def iterate_over_children(children_list):
                     event_method = getattr(original_mapping[target], trigger)
                     event_method(fn=fn, **dependency)
 
+            # Allows some use of Interface-specific methods with loaded Spaces
```

Comment:
Instead of implementing Interface specific things in Blocks, can you overwrite __call__ in Interface?

Reply:
You can call super().__call__ where necessary

Reply:
This code is in `from_config()` not `__call__()`

Reply:
In this case, I don't think it's actually appropriate to put it in Interface. Because it's actually a `Blocks` that gets created, not an `Interface`. This additional hack simply lets the `Blocks` be used in some ways that `Interfaces` are used -- it's more or less an experimental thing, not necessarily something we want to support fully right now.